### PR TITLE
AFJ Interop: Inline value of recipient keys in ddo

### PR DESCRIPTION
**Pull request by Patrik-Stas**
Signed-off-by: Patrik Stas <patrik.stas@absa.africa>

### CODE REVIEW

File: aries_vcx/src/messages/connection/request.rs [hunk 1]
```diff
@@ -69,7 +69,7 @@ pub mod unit_tests {
             label: _label(),
             connection: ConnectionData {
                 did: _did(),
-                did_doc: _did_doc_vcx_new(),
```
Comment (mirgee) [inline aries_vcx/src/messages/connection/request.rs:72]: Is this function still used somewhere?

File: aries_vcx/src/did_doc/mod.rs [hunk 2]
```diff
@@ -376,6 +375,24 @@ pub mod test_utils {
         }
     }
 
+    pub fn _did_doc_vcx_new_inlined_recipient_keys() -> DidDoc {
```
Comment (mirgee) [inline aries_vcx/src/did_doc/mod.rs:378]: Isn't `_vcx_new_` pleonasmic? How about `_did_doc_inlined_recipient_keys`?

**Review by mirgee — CHANGES_REQUESTED** [review]

File: aries_vcx/src/messages/connection/request.rs (hunk 1 above)
Comment (Patrik-Stas) [reply aries_vcx/src/messages/connection/request.rs:72]: still has several usages

Comment (Patrik-Stas) [reply aries_vcx/src/messages/connection/request.rs]: removed

File: aries_vcx/src/did_doc/mod.rs (hunk 2 above)
Comment (Patrik-Stas) [reply aries_vcx/src/did_doc/mod.rs:378]: renamed

**Pull request merged**